packages/rlp/CHANGELOG.md

```diff
@@ -55,7 +55,7 @@ PRs included in this release:
 
 Source files from the `src` folder are now included in the distribution build, see PR [#97](https://github.com/ethereumjs/rlp/pull/97). This allows for a better debugging experience in debug tools like Chrome DevTools by having working source map references to the original sources available for inspection.
 
-[v2.2.7]: https://github.com/ethereumjs/rlp/compare/v2.2.6...v2.2.7
+[v2.2.7]: https://github.com/ethereumjs/ethereumjs-monorepo/compare/rlp@2.2.7...rlp@2.2.6
```

Comment:
These URLs are not accessible - I think for legacy releases (so the ones not in the monorepo) they should link back to the (soon-to-be) archived repo of RLP.

Reply:
I haven't pushed the tags yet, I wasn't sure if I should do that now or after merging

Reply:
this one is in the wrong order though :)